client/my-sites/domains/domain-management/dataviews/dataviews-fields/domain-field.tsx

```diff
@@ -1,22 +1,35 @@
-import { PartialDomainData } from '@automattic/data-stores';
+import { JobStatus, PartialDomainData } from '@automattic/data-stores';
 import { PrimaryDomainLabel } from '@automattic/domains-table';
+import { useTranslate } from 'i18n-calypso';
 import { useDomainsDataViewsContext } from '../use-context';
 
 interface Props {
 	domain: PartialDomainData;
 	isAllSitesView: boolean;
 	selectedFeature?: string;
+	completedJobs: JobStatus[];
 }
 
-const DomainField = ( { domain: partialDomain, isAllSitesView }: Props ) => {
+const hasFailedJobs = ( domain: PartialDomainData, completedJobs: JobStatus[] ) => {
+	return completedJobs.filter( ( job ) => job.failed.includes( domain.domain ) ).length > 0;
```

Comment:
```suggestion
	return completedJobs.some( job => job.failed.includes( domain.domain ) );
```
This will return immediately once it finds something rather than completing the loop with the filter

Reply:
Makes sense, updated here: https://github.com/Automattic/wp-calypso/pull/100666/commits/98cdc205f52c333589a2af3de0b53ec8ed42d92d